src/main/resources/locale_fr_FR.properties

```
@@ -217,36 +217,40 @@ command.repsettings.info.message.title=Paramètres de réputation
-command.roles.sub.refresh.started=Le rafraîchissement des rôles a commencé.
-command.roles.sub.remove=Remplacer un rôle de réputation.
-command.roles.sub.remove.arg.role=Le rôle qui ne doit plus être un rôle de réputation.
-command.roles.sub.remove.notARepRole=Ce rôle n'est pas un rôle de réputation.
-command.roles.sub.remove.removed=Remove le rôle %ROLE%.
-command.roles.sub.removeDonor=Supprimer un rôle de donneur
-command.roles.sub.removeDonor.arg.role=Le rôle de donateur à supprimer.
-command.roles.sub.removeDonor.remove=Suppression du rôle de donneur %ROLE%
-command.roles.sub.removeReceiver=Supprimer un rôle de récepteur
-command.roles.sub.removeReceiver.arg.role=Le rôle de récepteur à supprimer.
-command.roles.sub.removeReceiver.remove=Suppression du rôle de récepteur %ROLE%.
-command.roles.sub.stackRoles=Autoriser le cumul des rôles
-command.roles.sub.stackRoles.arg.stack=Set to "true" to stack.
-command.roles.sub.stackRoles.notStacked=L'utilisateur n'aura que le rôle le plus proche de sa réputation.
-command.roles.sub.stackRoles.stacked=Les rôles seront empilés. Les utilisateurs auront tous les rôles avec moins que leur réputation.
+command.roles.info.message.donorroles=Rôles de donneur
+command.roles.info.message.receiverroles=Rôles de receveur
+command.roles.info.message.reputationrole=Rôles de réputation
+command.roles.info.message.roleinfo=Info sur les rôles
+command.roles.list.description=Lister les rôles de réputation.
+command.roles.refresh.description=Mettre à jour les rôles des utilisateurs
+command.roles.refresh.message.finished=Mise à jour du rôle de l'utilisateur effectuée.
+command.roles.refresh.message.running=Une mise à jour du rôle de l'utilisateur est déjà en cours.
+command.roles.refresh.message.started=Le rafraîchissement des rôles a commencé.
+command.roles.remove.description=Remplacer un rôle de réputation.
+command.roles.remove.message.noreprole=Ce rôle n'est pas un rôle de réputation.
+command.roles.remove.message.removed=Remove le rôle %ROLE%.
+command.roles.remove.role.description=Le rôle qui ne doit plus être un rôle de réputation.
+command.roles.removedonor.description=Supprimer un rôle de donneur
+command.roles.removedonor.role.description=Le rôle de donateur à supprimer.
+command.roles.removereceiver.description=Supprimer un rôle de récepteur
+command.roles.removereceiver.message.remove=Suppression du rôle de récepteur %ROLE%.
+command.roles.removereceiver.role.description=Le rôle de récepteur à supprimer.
+command.roles.removeronor.message.remove=Suppression du rôle de donneur %ROLE%
+command.roles.stackroles.description=Autoriser le cumul des rôles
+command.roles.stackroles.message.notStacked=L'utilisateur n'aura que le rôle le plus proche de sa réputation.
+command.roles.stackroles.message.stacked=Les rôles seront empilés. Les utilisateurs auront tous les rôles avec moins que leur réputation.
+command.roles.stackroles.stack.description=Set to "true" to stack.
```

Looks like we had missed this missing translation

DeepL suggestion:
```suggestion
command.roles.stackroles.stack.description=Mettre à "true" pour empiler.
```